src/middlewares/request.middleware.js

```diff
@@ -82,6 +82,14 @@ const requestMidd = {
     next();
   },
 
+  async chekIfParentExist(req, res, next) {
+    if (req.body.parent) {
+      const isFound = await CommentService.getComment(req.body.parent);
+      if (!isFound) return sendResult(res, 404, 'Parent comment does not exist');
+    }
+    next();
+  },
```

Comment:
If a comment does not have a parent, then it is a parent comment. We should also accept comments without parents